src/tutorial/output.md

```diff
@@ -1,15 +1,140 @@
 # Output
 
+## Printing "Hello World"
+
+```rust
+println!("Hello World");
+```
+
+Well, that was easy.
+Great, onto the next topic.
+
+## Using println
+
+You can pretty much print all the things you like
+with the `println!` macro.
+This macro has some pretty amazing capabilities,
+but also a special syntax.
+It expects you to write,
+as a first parameter,
+a literal string that with placeholders,
+followed by the data you want to put into these placeholders
+as further arguments.
+
+For example:
+
+```rust
+let x = 42;
+println!("My lucky number is {}.", x);
+```
+
+will print
+
+```console
+My lucky number is 42.
+```
+
+The curly braces (`{}`) in the string above is one of these placeholders.
+This is the default placeholder type
+that tries to print the given in a human readable way.
+For numbers and strings this works very well,
+but not all types can do that.
+This is why there is also a "debug representation",
+that you can get by filling the braces of the placeholder like this: `{:?}`.
```

Comment:
There are many more od these, maybe we want to link to the section in the book where they're explained?

Reply:
there's a link to the std::fmt docs in the very next aside